### ISSUE TITLE
TypeError: Object.keys(...).map(...).flat is not a function

### ISSUE THREAD

**Issue by arthurlogilab** [tomorrowstudio/generator-gatsby#9]
```
events.js:174
      throw er; // Unhandled 'error' event
      ^

TypeError: Object.keys(...).map(...).flat is not a function
    at module.exports.writing (/home/arthur/.npm-packages/lib/node_modules/generator-gatsby/generators/app/index.js:110:8)
    at Object.<anonymous> (/home/arthur/.npm-packages/lib/node_modules/generator-gatsby/node_modules/yeoman-generator/lib/index.js:402:23)
    at /home/arthur/.npm-packages/lib/node_modules/generator-gatsby/node_modules/run-async/index.js:25:25
    at new Promise (<anonymous>)
    at /home/arthur/.npm-packages/lib/node_modules/generator-gatsby/node_modules/run-async/index.js:24:19
    at self.env.runLoop.add.completed (/home/arthur/.npm-packages/lib/node_modules/generator-gatsby/node_modules/yeoman-generator/lib/index.js:403:9)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
Emitted 'error' event at:
    at Immediate.setImmediate (/home/arthur/.npm-packages/lib/node_modules/generator-gatsby/node_modules/yeoman-generator/lib/index.js:415:18)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
```

sorry, not sure this is a problem here or elsewhere

**Comment by seaneking** [comment]
Duplicate of #8 

Sorry this is an issue on Node <11. I’ll shim this function ASAP.

**Comment by seaneking** [comment]
Fixed in `1.4.3`
